openwrt_luci_rpc/openwrt_luci_rpc.py

```diff
@@ -62,21 +64,39 @@ def _determine_if_legacy_version(self):
         :return: tuple, with True if legacy and the URL to
                 use to lookup devices
         """
+
+        # NEW METHOD TO DETERMINE OPENWRT VERSION EXACTLY
+        # TODO: Check as non-root user
+
+        log.info("getting openwrt version")
+
+        rcp_sys_version_call = Constants.\
+            LUCI_RPC_SYS_PATH.format(self.host_api_url), "exec"
+
+        try:
+            content = self._call_json_rpc(*rcp_sys_version_call,
+                                          "cat /etc/openwrt_version")
```

Comment:
This line is failing flake8

```
openwrt_luci_rpc/openwrt_luci_rpc.py:78:42: E999 SyntaxError: only named arguments may follow *expression
```

Please run `tox` on your branch and get a pass before pushing.

Reply:
I also ran tox but it passed successfully

`
OK
flake8 inst-nodeps: /home/edoardo/Code/openwrt-luci-rpc/.tox/.tmp/package/1/openwrt-luci-rpc-1.0.7.zip
flake8 installed: certifi==2019.6.16,chardet==3.0.4,Click==7.0,entrypoints==0.3,flake8==3.7.7,idna==2.8,mccabe==0.6.1,openwrt-luci-rpc==1.0.7,pycodestyle==2.5.0,pyflakes==2.1.1,requests==2.21.0,urllib3==1.24.3
flake8 run-test-pre: PYTHONHASHSEED='4094220474'
flake8 run-test: commands[0] | flake8 openwrt_luci_rpc
___________________________________________________________________________________________ summary ___________________________________________________________________________________________
  py37: commands succeeded
  flake8: commands succeeded
  congratulations :)`

Anyway, I expanded the parameters so that that error disappears